xbmc/settings/DisplaySettings.cpp

```diff
@@ -792,6 +792,26 @@ void CDisplaySettings::SettingOptionsMonitorsFiller(SettingConstPtr setting, std
     }
     list.push_back(std::make_pair(monitors[i], monitors[i]));
   }
+#elif defined(HAVE_WAYLAND)
```

Comment:
OK, so: If you start with the projector switched off and then go to system settings, does it suddenly open the "Do you want to keep this resolution?" window?